src/main/java/org/netbeans/apitest/SigtestCheck.java

```diff
@@ -97,22 +97,28 @@
     private File report;
     @Parameter(defaultValue = "true", property = "sigtest.fail")
     private boolean failOnError;
+    /**
+     * By default (with value set to <true>) the project build directory as well as the project dependencies are added to the sigtest classpath.
```

Comment:
Does the default (true) match the current behavior?  From the issue description, I believe the answer is yes but wanted to verify.

Reply:
That's right.. the current behavior acts as if this is `true`